libraries/botbuilder-core/botbuilder/core/teams/teams_activity_handler.py

```diff
@@ -709,7 +709,7 @@ async def on_teams_team_unarchived(  # pylint: disable=unused-argument
 
     async def on_teams_members_added_dispatch(  # pylint: disable=unused-argument
         self,
-        members_added: [ChannelAccount],
+        members_added: ChannelAccount,
```

Comment:
Why the parameter type changed from a list to a single value? Please provide some context.